modules/serverless-target-utilization.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * /serverless/autoscaling/serverless-autoscaling-concurrency.adoc
+
+[id="serverless-target-utilization_{context}"]
+= Concurrency target utilization
+
+This value specifies the percentage of the concurrency limit that is actually targeted by the autoscaler. This is also known as specifying the _hotness_ at which a replica runs, which enables the autoscaler to scale up before the defined hard limit is reached.
+
+For example, if the `containerConcurrency` annotation value is set to 10, and the `targetUtilizationPercentage` value is set to 70 percent, the autoscaler creates a new replica when the average number of concurrent requests across all existing replicas reaches 7. Requests numbered 7 to 10 are still be sent to the existing replicas, but this allows for additional replicas to be started in anticipation of being needed when the `containerConcurrency` annotation limit is reached.
```

Comment:
"are still be sent to" ...should this be "are still to be sent to" ? Sorry if my English is limited here.

Reply:
Fixed it, good catch, thanks!